documents/integrate-with-hotwax/api/carriers/hotwax-carrier-api.md

```diff
@@ -0,0 +1,195 @@
+      "fragile": "true",
+      "description": "Puma Shoes"
+    }
+  ],
+  "countryId": "Guatemala",
+  "countPieces": "4",
+  "currencyUomId": "USD",
+  "cod": false,
+  "collected": "false",
+  "validShipmentTotal": "141.99",
+  "weightAmount": "4",
+  "identification": "null",
+  "facilityIdentification": "",
+  "orderNumber": "1088",
+  "ticketNumber": "GTP"
+}
+```
+
+### Request Parameters
+
+| Parameter                    | Type   | Description                                              |
+|------------------------------|--------|----------------------------------------------------------|
+| `shippingGatewayConfigId`    | string | Identifier for the shipping gateway configuration.      |
+| `carrierPartyId`             | string | Identifier for the carrier party.                       |
+| `dateOfSale`                 | string | Date of sale in `YYYY-MM-DD` format.                   |
+| `originAddress`              | object | Address from where the shipment is originating.         |
+| `destAddress`                | object | Destination address for the shipment.                   |
+| `parcels`                   | array  | Array of parcel objects containing shipment details.     |
+| `countryId`                  | string | Country identifier.                                     |
+| `countPieces`                | string | Total number of pieces in the shipment.                 |
+| `currencyUomId`             | string | Currency unit of measure identifier.                    |
+| `cod`                        | boolean| Indicates if cash on delivery is applicable.            |
+| `collected`                  | boolean| Indicates if the shipment is collected.                 |
+| `validShipmentTotal`         | string | Total valid shipment amount.                             |
+| `weightAmount`               | string | Total weight of the shipment.                            |
+| `identification`             | string | Additional identification (can be null).                |
+| `facilityIdentification`      | string | Identifier for the facility (if applicable).            |
+| `orderNumber`                | string | Order number associated with the shipment.              |
+| `ticketNumber`               | string | Ticket number associated with the shipment.             |
+
+## Response Format
+
+The response from the carrier API will be in JSON format and can be of two types: one for multiple packages and another for a single package.
+
+### Response Structure for Multiple Packages
+
+```json
+{
+  "responseMap": {
+    "success": true,
+    "shippingLabelMap": {
+      "referenceNumber": "FD10922319",
+      "packages": [
+        {
+          "imageBytes": "base64_encoded_image_data",
+          "trackingIdNumber": "FD10922319"
+        },
+        {
+          "imageBytes": "base64_encoded_image_data",
+          "trackingIdNumber": "FD10922319"
+        }
+      ]
+    }
+  }
+}
+```
+
+### Response Structure for Single Package
```

Comment:
is there a response structure expected for multiple packages as well?

Reply:
The packages field in the response supports a list of maps. If there are multiple packages, they will be included in this list. Therefore, I removed the extra structure example from the response.